159356-1702-Extramural/capstone
# Feature/sync timer

Basic concept:
On round start or player displaying monopoly, server creates a timestamp + round/monopoly time and starts a setTimeout.  Server sends that timestamp to client, client starts the timer animation with that timestamp as the endpoint (has no actual timer itself).  When round time expires, each player gets a request to finish their turn. When monopoly time expires, monopoly owner gets a notification to close window, everyone else gets start turn and animation for round begins.

To test: 

- Start a game with {url}/startWithCards=1&dev_card=monopoly, move through setup and finish the first round (the first playing round has no timer at present to look around.. maybe look at an extended timer to allow looking around).
- Let the turns expire for a few turns(and check logs that timer is reporting times).  Have one player purchase dev card (set to monopoly) watch turns expire.
